src/builder/FFI.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.27;
+
+import {IAcrossFFI} from "src/interfaces/IAcrossFFI.sol";
+
+/**
+ * @title Foreign Function Interface (FFI) Helper
+ * @notice Defines the addresses of reserved FFIs and methods for calling them
+ */
+library FFI {
+    /* FFI Addresses (starts from 0xFF1000, FFI with 100 reserved addresses) */
+    address constant ACROSS_FFI_ADDRESS = address(0xFF1000);
```

Comment:
If we reserve 0x100 from `0xFF1000` then this should be 0xFF1100`, right? Unless we're reserving 0x1000 from from `0xFF0000` then that would align, but I don't think this maps with the comment.

Reply:
Updated the comment to reflect that there are 1000 reserved addresses